cockatrice/src/settings/gamefilterssettings.cpp

```diff
@@ -129,3 +129,47 @@ bool GameFiltersSettings::isGameTypeEnabled(QString gametype)
     QVariant previous = getValue("game_type/" + hashGameType(gametype), "filter_games");
     return previous == QVariant() ? false : previous.toBool();
 }
+
+void GameFiltersSettings::setShowOnlyIfSpectatorsCanWatch(bool show)
+{
+    setValue(show, "show_only_if_spectators_can_watch", "filter_games");
+}
+
+bool GameFiltersSettings::isShowOnlyIfSpectatorsCanWatch()
+{
+    QVariant previous = getValue("show_only_if_spectators_can_watch", "filter_games");
+    return previous == QVariant() ? true : previous.toBool();
```

Comment:
I think this default is a bit harsh, people most likely still want to join games even if they don't allow spectators.

Reply:
Switched to false